### PR TITLE
RamVasuthevan/create-frontend

### PR DESCRIPTION
**Pull request by RamVasuthevan**

### CODE REVIEW

**Comment by kyleyarwood** [comment]
I know it's a draft but LGTM lol

**Comment by RamVasuthevan** [comment]
Make sure that all times are EST

**Comment by RamVasuthevan** [comment]
I'm gonna be honest, I still think there's room for improvement in both the UI and code but I just want to get a POC done

File: website/script.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+async function displayAllRoutes() {
+    // Fetches and processes the ferry schedule, adjusting dates to America/Toronto timezone
+    const response = await fetch('data/schedule.json');
+    const data = await response.json();
+    const schedules = data.schedules;
+    const today = moment.tz('America/Toronto').format('YYYY-MM-DD');
+    const routeSelect = document.getElementById('ferryRoute');
+
+    schedules.forEach(schedule => {
+        if (moment(schedule.start).isSameOrBefore(today) && (schedule.end === null || moment(schedule.end).isSameOrAfter(today))) {
```
Comment (kyleyarwood) [inline website/script.js:16]: schedule.start and schedule.end will have to be updated to use the new model for start and end

File: website/script.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    const response = await fetch('data/schedule.json');
+    const data = await response.json();
+    const schedules = data.schedules;
+    const today = moment.tz('America/Toronto').format('YYYY-MM-DD');
+    const timesContainer = document.getElementById('ferryTimes');
+    timesContainer.innerHTML = `<p>Ferries within the next hour are highlighted</p>`;
+    timesContainer.classList.add('styledFerryTimes');
+
+    schedules.forEach(schedule => {
+        if (moment(schedule.start).isSameOrBefore(today) && (schedule.end === null || moment(schedule.end).isSameOrAfter(today))) {
```
Comment (kyleyarwood) [inline website/script.js:47]: same thing here for schedule.start and schedule.end

**Review by kyleyarwood — APPROVED** [review]
pending changes to start and end usage

**Comment by RamVasuthevan** [comment]
We should be thinking of each route entry as occurring on a day at a time (in est)

**Pull request merged**